attest/ake/src/event.rs

```diff
@@ -23,8 +23,8 @@ where
 {
     /// This is the local node's identity key
     pub(crate) local_identity: KexAlgo::Private,
-    /// This is the local node's ias report.
-    pub(crate) attestation_evidence: EvidenceKind,
+    /// This is the local node's attestation evidence.
+    pub(crate) dcap_evidence: DcapEvidence,
```

Comment:
This only needs to be DcapEvidence, it prevents looking for an invalid state.

I renamed the field to `dcap_evidence` as well to make it a bit easier in the usage locations when it's DcapEvidence vs EvidenceKind.